beancount_import/source/schwab_csv.py

```diff
@@ -343,7 +350,6 @@ def get_meta(self) -> Meta:
     def get_narration_prefix(self) -> str:
         raise NotImplementedError()
 
-
```

Comment:
Nit: two lines between top level classes is typical and intentional formatting, please restore this blank line.